core/services/chainlink/application.go

```diff
@@ -351,6 +350,11 @@ func (app *ChainlinkApplication) Start() error {
 		}
 	}
 
+	// EthBroadcaster needs to be started after GasUpdater (in subservices)
+	if err := app.EthBroadcaster.Start(); err != nil {
+		return err
+	}
+
```

Comment:
Why not move it into subservices and/or remove it from the app?

Reply:
app.EthBroadcaster.Trigger() is used in tests, so can't easily remove it

```
	// trigger allows other goroutines to force ethBroadcaster to rescan the
	// database early (before the next poll interval)
```

Reply:
I think we could still make it a StartCloser and move it into `subservices` instead of manually start/stopping.

Then as @j16r suggested, it would be codified by its index in the slice. What do you think?

Reply:
cool, just did